src/preferences/broadcastsettings.cpp

```diff
@@ -199,12 +191,17 @@ void BroadcastSettings::applyModel(BroadcastSettingsModel* pModel) {
     // TODO(Palakis): lock both lists against modifications while syncing
 
     // Step 1: find profiles to delete from the settings
-    for(const auto& actualProfile : m_profiles) {
-        QString profileName = actualProfile->getProfileName();
+    for(auto actualProfile = m_profiles.begin(); actualProfile != m_profiles.end();) {
```

Comment:
we could also use `m_profiles.values()` here, but using `erase()` is slightly more efficient.

Reply:
Even as a non-native speaker I suppose the term "actual" is wrong here. We should replace it. Since it is an iterator you may use "iterProfile" or whatever. I don't like type prefixes, but in this case it might be helpful to distinguish the iterator from the actual type, i.e. "removedProfile". Maybe "missingProfile" or "orphanedProfile" would be a more appropriate naming for this variable, because this is what we actually know about it before we are going to delete it.

Please add a space after the keyword _for_ to distinguish it from a function call.